test-cases/gemini/gemini-basic-3h-ics.yaml

```diff
@@ -3,22 +3,26 @@ n_db_nodes: 3
 n_test_oracle_db_nodes: 1
 n_loaders: 1
 n_monitor_nodes: 1
-instance_type_db: 'i3.large'
+instance_type_db: 'i3.4xlarge'
+instance_type_loader: 'c5.2xlarge'
 
 user_prefix: 'ics-gemini-basic-3h'
 ami_db_scylla_user: 'centos'
 
 # gemini
 # cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -m mixed -f
 # the below cmd runs about 3 hours
-gemini_cmd: "gemini -d --duration 10800s --warmup 1800s -c 50 -m mixed -f --non-interactive \
+gemini_cmd: "gemini -d --duration 10800s --warmup 1800s -c 25 -m mixed -f --non-interactive \
 --cql-features normal --table-options \"compaction={'class': 'IncrementalCompactionStrategy'}\" \
---max-mutation-retries 5 --max-mutation-retries-backoff 500ms \
---async-objects-stabilization-attempts 5 --async-objects-stabilization-backoff 500ms "
+--max-mutation-retries 25 --max-mutation-retries-backoff 10s \
```

Comment:
these number are pretty big. I propose setting it to 10 and 2s (unless you tested and proven current ones are right)

Reply:
For that moment this numbers give most stable results, especially for MV and SI operations